dask/dataframe/io/tests/test_parquet.py

```diff
@@ -274,6 +274,20 @@ def test_read_glob(tmpdir, write_engine, read_engine):
     assert_eq(ddf, ddf2)
 
 
+@write_read_engines()
+def test_gather_statistics_false(tmpdir, write_engine, read_engine):
```

Comment:
This isn't meant as a blocking comment, just an FYI. `pytest` has a [`tmp_path` fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture) which yields a `pathlib.Path` object that `to_parquet` / `read_parquet` should be able to understand. Though the `str` conversion you have here is totally correct and used in other tests throughout this module

Reply:
That's a good point - We should probably make sure that we are handling pathlib.Path objects in at least some tests.